cmd/srcd/cmd/prune_test.go

```diff
@@ -53,8 +51,8 @@ func (s *PruneTestSuite) TearDownTest() {
 	os.RemoveAll(s.testDir)
 }
 
-func (s *PruneTestSuite) TestRunningContainers() {
-	require := s.Require()
+func (s *PruneTestSuite) testRunningContainers(require *require.Assertions, pruneCmd func() (*bytes.Buffer, error)) {
```

Comment:
Maybe it would be simpler to understand if we use `args ...string` instead of `pruneCmd func() (*bytes.Buffer, error)`.
Or it could be a `withImages bool`.
WDYT?

Reply:
+1 to `withImages` flag.

Reply:
yeah much better, [here](https://github.com/src-d/engine/pull/405/commits/f18d31fb8a9ae8bd93c8a7e30d64a21b7666f956).